### ISSUE TITLE
Buddys duplicated in 'Buddies' List and user defined Group list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#6]
```
What steps will reproduce the problem?
1. Log into MSN using pidgin libmsn protocol. Have a user-defined group
created with buddies populated in it.
2. Log out of MSN
3. Log into MSN using WLM protocol

What is the expected output? What do you see instead?
The groups should be the same as what was on MSN but instead I will have a
buddy in my 'Buddies' list and also the same buddy in my user-defined list.
Originally the buddy was only in the user-defined list and not in the
Buddies list.

What version of the product are you using? On what operating system?
pidgin 2.3.1 compiled from src, Xubuntu, 7.10

Please provide any additional information below.


```

Original issue reported on code.google.com by `craigwharding` on 6 Feb 2008 at 2:09

**Comment by GoogleCodeExporter** [comment]
```
Sorry. dup of #2:
http://code.google.com/p/msn-pecan/issues/detail?id=2&can=1
```

Original comment by `craigwharding` on 6 Feb 2008 at 2:11

**Comment by GoogleCodeExporter** [comment]
```
Yes, dup.

I've just released 0.0.8, please try again.
```

Original comment by `felipe.contreras` on 6 Feb 2008 at 6:16
- Changed state: **Duplicate**
